security/certificate_types_descriptions/monitoring-and-cluster-logging-operator-component-certificates.adoc

```diff
@@ -0,0 +1,14 @@
+[id="cert-types-monitoring-and-cluster-logging-operator-component-certificates"]
+= Monitoring and cluster logging Operator component certificates
+include::modules/common-attributes.adoc[]
+:context: cert-types-monitoring-and-cluster-logging-operator-component-certificates
+
+toc::[]
+
```

Comment:
I would put this intro content under a heading called Expiration to be consistent with other sections